src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from 'react';
+import { PersonRow } from '../PersonRow';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { useParams } from 'react-router-dom';
+
+export const PeopleTable = () => {
+  const { slug } = useParams();
+  const normalizedSlug = slug || null;
+
+  const [loading, setLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    getPeople()
+      .then(response => setPeople(response))
+      .catch(() => setErrorMessage('Something went wrong'))
+      .finally(() => setLoading(false));
+  }, []);
+
+  return (
+    <div className="box table-container">
+      {loading && <Loader />}
```

Comment:
```suggestion
      {isLoading ? <Loader /> : (everything else)}
```
You can use ternary operator here to improve readability